crates/sui-move-build/src/lib.rs

```diff
@@ -540,10 +568,8 @@ impl CompiledPackage {
         })
     }
 
-    pub fn verify_unpublished_dependencies(
-        &self,
-        unpublished_deps: &BTreeSet<Symbol>,
-    ) -> SuiResult<()> {
+    pub fn verify_unpublished_dependencies(&self) -> SuiResult<()> {
+        let unpublished_deps = &self.dependency_ids.unpublished;
```

Comment:
Simplified here because we call this after we create the `CompiledPackage` type, so `dependency_ids` is up-to-date and we do not need to pass it in.